components/shopify_partner/sources/new_app_charges.js

```diff
@@ -0,0 +1,67 @@
+const shopify = require("../shopify_partner.app.js");
+const getAppTransactions = require("../queries/getAppTransactions");
+
+module.exports = {
+  key: "shopify_partner-new-app-charges",
+  name: "New App Charges",
+  type: "source",
+  version: "0.0.1",
+  description:
+    "Emit new events when new app charges made to your partner account.",
+  props: {
+    shopify,
+    db: "$.service.db",
+    createdAtMin: {
+      type: "string",
```

Comment:
Are these props required or optional?

Reply:
They are optional, but I thought the component docs mentioned that the `optional` param was implicitly true, but I can make it explict.